### ISSUE TITLE
NPM package installation fails

### ISSUE THREAD

**Issue by XmlmXmlmX** [javanto/civem.js#8]
There is no official NPM package: https://www.npmjs.com/search?q=civem

When installing via git URL (`npm install https://github.com/javanto/civem.js.git -save`), the package folder is empty (there is only a README.md).

**Comment by XmlmXmlmX** [comment]
I've added civim to npm registry: https://www.npmjs.com/package/civem

**Comment by hleinone** [comment]
@XmlmXmlmX Thanks! Is there still something to be done to fix the issue?

**Comment by XmlmXmlmX** [comment]
No, the package is still empty. The package.json contains some placeholders. They look like mustache or handlebar:

```
"files": [
    "dist/{{= it.name}}.js", 
    "dist/{{= it.name}}.min.js"
  ],
  "build": {
    "directory": "dist",
    "final_name": "{{= it.name}}",
    "unique_name": "{{= it.name}}-{{= it.version}}"
},
```

How does this work inside a package.json? I just know that environment variables can be used.

**Comment by XmlmXmlmX** [comment]
Should I add you as owner to the registered NPM package at npmjs.com?

**Comment by hleinone** [comment]
If I remember correctly jake replaces those when doing a release, so in the release package.json those would be replaced. I'm not sure if that's the best option in 2016, haven't really done any JS in the past few years. 😁 Sure, go ahead and add me as the owner. And thanks again!

**Comment by XmlmXmlmX** [comment]
Well, in 2016 the common way is Grunt or Gulp, :) I have never worked with Jake.
With Jake it hasn't work.

**Comment by XmlmXmlmX** [comment]
work in progress ...

**Comment by extempl** [comment]
Jake/grunt/gulp - doesn't matters. `package.json` just should be valid json with correct data.
Filenames, versions, etc, should be set on build and committed to repo. If you need a template you could create file like `package-template.json` and build `package.json` from it.

This is also applies to compiled files - these are should be present in the repo.
